### DISCUSSION TITLE
minimumReleaseAge includes  ages "new"  and ?

### DISCUSSION THREAD

**Discussion by cubasuarez** [renovatebot/renovate#26011] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

With the following configuration:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "schedule": [
    "before 9am on Monday"
  ],
  "assignees": [],
  "reviewers": [],
  "packageRules": [
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["patch"],
      "groupName": "all patch dependencies",
      "groupSlug": "all-patch",
      "minimumReleaseAge": "15 days"
    }
  ]
}

```

I'm surprised to see jacoco and  eclipse-temurin showingup in the PR "Update all patch dependencies".
![image](https://github.com/renovatebot/renovate/assets/527357/7e11df8b-62d2-48e7-8d97-deb7f41d2744)

I was expecting them to show up in separate PR's. But I might have missed some configuration option.
 

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
Renovate and Merge Confidence operate independently, and may occasionally diverge.

Merge Confidence badges fall back to "NEW" for age of any version it hasn't indexed yet. Renovate does not read that value and instead bases its behavior on what it retrieves, So for example Merge Confidence won't return data about a private package because it has no knowledge about it, while Renovate could happily determine the release age so long as you've configured it with credentials.

Is jacoco a private npm dependency? It's not found at https://www.npmjs.com/package/jacoco

**Reply by cubasuarez** [reply]
This may explain it. These are the jacoco links:
[jacoco](http://jacoco.org/) ([source](https://togithub.com/jacoco/jacoco))

Is there any flag to set to tell renovate to not include the one where a version cannot be determined (as it will be offered as a separate pull request anyway if I'm not misunderstanding)?

**Reply by rarkins** [reply]
From the looks of your screenshot, Renovate has found and updated the dependency. To Renovate, it all looks good. It then adds an image URL which is loaded from the merge confidence servers. The Merge Confidence servers aren't aware of the package and/or release, so have to respond with _something_.

I don't understand why you say "as it will be offered as a separate.." because in your screenshot it's in the same PR
